.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,83 @@
+name: allcontributors-auto-detect
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Tributors Update      
+        uses: con/tributors@0.0.15
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # Single text list (space separated) of parsers, leave unset to auto-detect
+          parsers: unset
+
+          # Update lookup with GitHub metadata
+          update_lookup: github
+
+          # Skip these users (example)
+          skip_users: manbat
+
+          # INFO, DEBUG, ERROR, WARNING, etc.
+          log_level: DEBUG
+
+          # If files already exist and an init is done, force overwrite
+          force: true
+
+          # the minimum number of contributions required to add a user
+          threshold: 1
+
+      - name: Checkout New Branch
```

Comment:
Why this step is not a part of the `con/tributors@0.0.15` action?  It seems to have nothing datalad specific.

Reply:
The action is well scoped to only generate updated content. It's up to the user to do what they want with it (e.g., push to a new branch, add somewhere else, etc.)

Reply:
but then everyone needs to copy/paste this lengthy code snippet etc... well, ok for now I guess